libs/tokens/src/utils/tokenWithLogoFromToken.ts

```diff
@@ -0,0 +1,7 @@
+import { TokenWithLogo } from '@cowprotocol/common-const'
+import { Token } from '@uniswap/sdk-core'
+import { TokenInfo } from '@uniswap/token-lists'
+
+export function tokenWithLogoFromToken(token: Token | TokenInfo, logoUrl?: string): TokenWithLogo {
+  return new TokenWithLogo(logoUrl, token.chainId, token.address, token.decimals, token.symbol, token.name)
+}
```

Comment:
Nitpick: feels like this should be part of the TokenWithLogo class. 

Something like:

```
static fromToken(token: Token, logoUrl: string) => TokenWithLogo
```

Reply:
This is a great idea!
https://github.com/cowprotocol/cowswap/pull/3201/commits/74003451f9cb93c14ef6657147dd8115f7938415